crates/forge/tests/integration/available_gas.rs

```diff
@@ -37,7 +37,7 @@ fn available_gas_exceeded() {
     assert_case_output_contains!(
         result,
         "keccak_cost",
-        "Test cost exceeded the available gas"
+        "Test cost exceeded the available gas. Gas consumed: ~11"
```

Comment:
[total nit] But I liked **Consumed gas: ~11** more 😄 something about the word `gas` being closer to the value, like in passed tests